### DISCUSSION TITLE
Engines choice option

### DISCUSSION THREAD

**Discussion by ghost** [hearchco/frontend#485] [Ideas]
Hi, the hearch.co page doesn’t seem to have an option for selecting the search engines that you want use. Would it be possible to implement it? Something like what SearXNG already does.

**Comment by aleksasiriski** [comment]
It's planned to make custom user categories (lenses / goggles / filters) but currenctly this isn't implemented, a plan on how this could be achieved would be a very welcome contribution (as well as the code itself for both frontend and agent of course).
